src/main/java/org/elasticsearch/search/highlight/HighlightPhase.java

```diff
@@ -46,6 +45,7 @@
 public class HighlightPhase extends AbstractComponent implements FetchSubPhase {
 
     private final Highlighters highlighters;
+    private static final ImmutableList<String> sortedHighlighters = ImmutableList.of("fvh", "postings", "plain");
```

Comment:
rename to `STANDARD_HIGHLIGHTERS_BY_PRECEDENCE` or something? I would like to remark in the name that they are sorted by precedence and its only the standard highlighters, not custom ones (maybe a comment would do it too).